docs/src/conf.py

```diff
@@ -101,9 +102,34 @@ def autolog(message):
 
 build_python_version = ".".join([str(i) for i in sys.version_info[:3]])
 
+
+def _dotv(version):
+    result = version
+    match = re.match("^py(\d+)$", version)  # noqa: W605
```

Comment:
Could be a raw string instead of overriding the linter?

Reply:
https://www.flake8rules.com/rules/W605.html

Reply:
Done :+1: